streaming/kafka-confluent/README.md

```diff
@@ -0,0 +1,7 @@
+# Deploying a stateful Kafka cluster with Confluent operator on GKE
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/kubernetes-engine-samples&cloudshell_tutorial=cloudshell/tutorial.md&cloudshell_workspace=gke-kafka-confluent)
+
+This example shows how to build and deploy a stateful Kafka using [Kubernetes](https://kubernetes.io) and Day 1 & 2 operations such as Monitoring with [Cloud Monitor](https://cloud.google.com/monitoring) and disaster recovery with [Backup for GKE](https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/concepts/backup-for-gke).
+
+Follow this tutorial at [https://cloud.google.com/kubernetes-engine/docs/tutorials/stateful-workloads/confluent](https://cloud.google.com/kubernetes-engine/docs/tutorials/stateful-workloads/confluent)
```

Comment:
**Question:** This URL 404s — both on production and staging. Do you have a CL or a doc where this tutorial is being drafted?

Reply:
Nope, I just supposed which url could be used, I dont have any access to CLs

Reply:
@NimJay The docs CL is still in development by GKE iX, but the URL where the doc will land should be correct (https://cloud.google.com/kubernetes-engine/docs/tutorials/stateful-workloads/confluent). I've shared the draft doc with you separately, for your reference.

Reply:
Thank you, both!